### DISCUSSION TITLE
Question about updating terms

### DISCUSSION THREAD

**Discussion by ghost** [tosdr/tosdr.org#226] [Q&A] [answered]
Right now because of the Coppa law getting revised, a whole bucket load of terms and policies are getting updated. How are those updates gonna be carried over onto tosdr?
Will you need help from us in edit.tosdr to supply the updates and will we need to re annotate any policies that already got graded?

**Comment by Vinnl** [comment, marked as answer]
Thanks @kitcallie. Yes, when you go to a Document's page, you can click "Crawl Document" to fetch a new version of those terms again. Sentences with Annotations that are unchanged will remain present, whereas all Annotations on sentences that have changed will move back to "Pending". It would be great if people could go over the updated Documents to see whether new Annotations would be relevant.

![image](https://user-images.githubusercontent.com/4251/70391317-2a337d00-19d4-11ea-84c8-65ddebc59f7e.png)
